crates/solidity/testing/snapshots/bindings_output/built_ins/functions/generated/0.8.4-success.txt

```diff
@@ -0,0 +1,26 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
+    ╭─[input.sol:1:1]
+    │
+  1 │ contract BuiltInsTest {
+    │          ──────┬─────  
+    │                ╰─────── def: 1
+  2 │     function testRequire() public {
+    │              ─────┬─────  
+    │                   ╰─────── def: 2
+  3 │         require(true, "should always succeed");
+    │         ───┬───  
+    │            ╰───── refs: built-in, built-in
+    │ 
+  6 │     function testRevert() public {
+    │              ─────┬────  
+    │                   ╰────── def: 3
```

Comment:
the `revert()` call was not bound in this version. Is this expected?

Reply:
`revert` is a proper statement starting at 0.8.4. At least that's what the grammar rules say :)

Reply:
it might be a bug in the grammar. Let me double check it ..